lib/simplekiq/polling_worker.rb

```diff
@@ -0,0 +1,58 @@
+class PollingWorker
+  include Simplekiq::Worker
+
+  def perform(params)
+    @params = params
+    return if expired?
```

Comment:
Can this return something useful if the job is expired?
Even just returning:
```suggestion
    return :expired if expired?
```

Reply:
Yeah, but this is a background job, so the return value is not important. Where would be make this visible?